addons/microsoft_calendar/models/calendar.py

```diff
@@ -521,6 +521,11 @@ def _microsoft_values(self, fields_to_sync, initial_values={}):
         if 'location' in fields_to_sync:
             values['location'] = {'displayName': self.location or ''}
 
+        if not self.location and 'videocall_location' in fields_to_sync:
+            values['isOnlineMeeting'] = True
+        else:
+            values['isOnlineMeeting'] = False
+
```

Comment:
Hello @geflx ,

I have updated the code and added the onlineMeetingProvider so that the event will only create team meetings but creating events with team meetings requires the Microsoft 365 plan. and also you have to select teams meeting for business option on Outlook here is the reference : https://answers.microsoft.com/en-us/msteams/forum/all/why-is-outlook-calendar-only-showing-new-skype/88fa967c-6d63-4a73-81e0-cfc50a8c16f8

Thanks for your suggestion